src/cli-storage.ts

```diff
@@ -0,0 +1,19 @@
+import fs from "fs";
+import os from "os";
+
+const __confDir = os.homedir() + "/.baseplate";
+if (!fs.existsSync(__confDir)) {
+  fs.mkdirSync(__confDir);
+}
+
+export async function storageProvider() {
+  // @ts-ignore
+  const conf = await import("conf");
+  return new conf.default({
+    projectName: "baseplate.cloud",
+    configName: ".bpc",
+    fileExtension: "",
+    cwd: __confDir,
+    encryptionKey: "910b01ee3a166b1132a01735ef4e695",
```

Comment:
Is this safe to hard-code? The baseplate-cli repo is public

Reply:
this is temporary, ideally it should be a one time generated thing, these sort of encryptions is only there in case if user file system exposed, the clear text version of the token was not being stored in the file system